### ISSUE TITLE
Color contrast test

### ISSUE THREAD

**Issue by MathijsN** [MathijsN/the-web-is-for-everyone-interactive-functionality#18]

**Comment by MathijsN** [comment]
<img width="747" height="256" alt="Image" src="https://github.com/user-attachments/assets/0dad0725-b272-41b7-9eba-bba3b976c7a5" />

**Comment by MathijsN** [comment]
<img width="471" height="596" alt="Image" src="https://github.com/user-attachments/assets/ac9e7da0-763c-4f4a-b24e-b0d77b3aa2f7" />

**Comment by MathijsN** [comment]
Color contrast is failing, will be discussed with product owner
